mmedit/apis/restoration_video_inference.py

```diff
@@ -18,24 +18,25 @@ def pad_sequence(data, window_size):
     return data
 
 
-def restoration_video_inference(model, img_dir, window_size):
+def restoration_video_inference(model, img_dir, window_size, filename_tmpl):
```

Comment:
Have you changed all references to this function?

Reply:
Okay it seems so

Reply:
Yes, only `restoration_video_demo.py` uses this fuction.